checks/burrowsc/burrow.go

```diff
@@ -0,0 +1,236 @@
+package burrowsc
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+
+	"github.com/hootsuite/healthchecks"
+)
+
+// Burrow consumer group and partition statuses (https://github.com/linkedin/Burrow/wiki/http-request-consumer-group-status)
+const (
+	OK       = "OK"
```

Comment:
we don't all caps constants in go, pick "Ok" if it needs to be exported otherwise "ok"

Reply:
Was looking to be consistent with how we're defining status constants (https://github.com/hootsuite/healthchecks/blob/2b4d3cd9e9f66846ea47c2e1d276695a57cdd178/healthchecks.go#L31), but I can update this.